app/services/gplaces.js

```diff
@@ -1,8 +1,8 @@
 const axios = require("axios");
 
-module.exports = function() {
-  this.getPlaceById = (id)=>{
-    return new Promise((resolve, reject)=>{
+module.exports = () => {
```

Comment:
Wenn das zu ner arrow function geändert wird, muss dann das return this weg, oder kann da trotzdem bleiben?

Reply:
Arrow functions verhalten sich bei "this" anders als eine normale function. Höchstwahrscheinlich funktioniert das mit function so  nicht

Reply:
wie meinst du?